src/Manager.php

```
@@ -495,7 +495,17 @@ public static function formatError(Error $exception)
             'message' => $exception->getMessage(),
         ];
 
-        $locations = $exception->getLocations();
+        try {
+            $locations = $exception->getLocations();
+        } catch (Exception $whatever) {
+            /* Ignore the exception to avoid the following error:
```

This seems _very_ hacky... we’d need to investigate the cause of the error instead of blindly suppressing it

---

Unfortunately this exception is raised by `webonyx/graphql-php` itself and has been introduced by  webonyx/graphql-php@cf276340a4249fb426a6c4fa940fb0e5fcbd455c.

---

Yuck :( do you know if that’s been raised as an issue on the `webonyx/graphql-php` Github? I think if we suppress this, we run the risk of incorrectly suppressing other errors :/

---

A safer disgusting hack might be to do some text checking on the exception message so you can re-throw exceptions that aren't this one

---

> do you know if that’s been raised as an issue on the `webonyx/graphql-php` ...

I am pretty sure nothing has been raised because when looking for information I did not find anything. I would have opened that issue by myself but I don't know the caller code well enough.

---

Right so it’s a test only problem? That’s good

---

Yes. The bug has always been there and webonyx/graphql-php@cf276340a4249fb426a6c4fa940fb0e5fcbd455c exposed it.